docs/installation.rst

```diff
@@ -0,0 +1,125 @@
+Installation
+============
+
+This installation guide includes only the GeoCAT-viz installation and build instructions.
+Please refer to `GeoCAT Contributor's Guide <https://geocat.ucar.edu/pages/contributing.html>`_ for installation of
+the whole GeoCAT project.
+
+Installing GeoCAT-viz via Conda in a New Environment
+-----------------------------------------------------
+
+The easiest way to install GeoCAT-viz is using
+`Conda <http://conda.pydata.org/docs/>`_::
+
+    conda create -n geocat -c conda-forge -c ncar geocat-viz
+
+where :code:`geocat` is the name of a new conda environment, which can then be
+activated using::
+
+    conda activate geocat
+
+If you need to make use of other software packages, such as Jupyter
+with GeoCAT-viz, you may wish to install into your :code:`geocat`
+environment.  The following :code:`conda create` command can be used to create a new
+:code:`conda` environment that includes one of these additional commonly used Python
+packages pre-installed::
+
+    conda create -n geocat -c conda-forge -c ncar geocat-viz jupyter
+
+Alternatively, if you already created a conda environment using the first
+command (without the extra packages), you can activate and install the packages
+in an existing environment with the following commands::
+
+    conda activate geocat # or whatever your environment is called
+    conda install -c conda-forge jupyter
+
+Also, note that the Conda package manager automatically installs all required dependencies,
+meaning it is not necessary to explicitly install NumPy, Matplotlib, Cartopy, Xarray, Scikit-learn,
+etc. when creating an environment.
+
+If you are interested in learning more about how Conda environments work, please
+visit the `managing environments <https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html>`_
+page of the Conda documentation.
+
+Installing GeoCAT-viz in a Pre-existing Conda Environment
```

Comment:
I think everything is looking good overall and is building locally for me! However, for some reason bullet points 1 and 2 in this section and in "Updating via PyPI" do not have the grey code box around them like point 3 does. I tried messing around with some of the spacing and placements of colons, but it didn't seem to fix it. I thought I would bring it to your attention in case you know what the fix may be, but overall, I don't think it would be a major detriment if we can't fix this style/formatting issue